qiskit/extensions/standard/ch.py

```diff
@@ -19,23 +19,22 @@
 """
 import numpy as np
 
-from qiskit.circuit import Gate
+from qiskit.circuit import ControlledGate
 from qiskit.circuit import QuantumCircuit
 from qiskit.circuit import QuantumRegister
-from qiskit.extensions.standard.h import HGate
-from qiskit.extensions.standard.cx import CnotGate
-from qiskit.extensions.standard.t import TGate
-from qiskit.extensions.standard.t import TdgGate
-from qiskit.extensions.standard.s import SGate
-from qiskit.extensions.standard.s import SdgGate
+import qiskit.extensions.standard.h as h
+import qiskit.extensions.standard.cx as cx
+import qiskit.extensions.standard.t as t
+import qiskit.extensions.standard.s as s
```

Comment:
why change these import styles? i don't particularly care about the style but it makes it inconsistent with the other files in this directory.

Reply:
this was remnant of old change. reverted.